bigbluebutton-html5/imports/ui/components/poll/live-result/component.jsx

```diff
@@ -57,7 +57,7 @@ class LiveResult extends PureComponent {
       ? [...users, ...responses.map(u => u.userId)]
       : [...users];
 
-    userAnswers = userAnswers.map(id => getUser(id))
+    userAnswers = userAnswers.map(id => Service.getUser(id))
```

Comment:
Shouldn't this Service call not be in the component because it won't be reactive? With this there's a weird edge case where someone can log out and still look like it's waiting for a response.